syntax = "proto3";

package ringvote;

import "question.proto";

message Response {
    message Field {
        oneof field {
            int64 int_field = 1;
            double decimal_field = 2;
            string string_field = 3;
            bytes bytes_field = 4;
        }
    }

    optional ringvote.QuestionType type = 1;
    repeated Field fields = 2;
}
